bears/documentation/DocumentationStyleBear.py

```diff
@@ -134,6 +134,11 @@ def run(self, filename, file, language: str,
                 new_metadata, doc_comment.docstyle_definition,
                 doc_comment.marker, doc_comment.indent, doc_comment.position)
 
+            if file[doc_comment.range.start.line - 2] != '\n':
```

Comment:
This will now add paddings for all languages. Is this what we want? Don't you want to define this in the style definitions?